scripts/release_notes.rb

```diff
@@ -7,57 +7,91 @@
 require 'uri'
 require 'net/http'
 require 'json'
-require 'pry-byebug'
 
 URL = 'https://projects.theforeman.org'
 
-# Check out installer modules
-# Compare packages in nightly vs release and retag dependencies
-
-# Create new release via API call
-# Create new search by open + release field
-
-unless @current_release_id = ARGV[0]
-  puts "Usage: #{$0} RELEASE_ID"
-  puts "Hint: 1.16.0 is 240"
+unless ARGV.size == 2
```

Comment:
Got an example of what this file generates?

Reply:
https://github.com/theforeman/theforeman.org/pull/1144/files#diff-66a06a0e26dd946fb2dfb39ace1e11c9